concrete/themes/dashboard/view.php

```diff
@@ -25,38 +25,36 @@
     </div>
 
     <?php
-    // We output the DIV even if it's empty because some pages might add to it via javascript ?>
+    // We output the DIV even if it's empty because some pages might add to it via javascript?>
     <div class="ccm-search-results-breadcrumb <?= (isset($breadcrumbClass)) ? $breadcrumbClass : ''; ?>">
 
     <?php if (isset($breadcrumb) && (!empty($breadcrumb))) { ?>
         <ol class="breadcrumb">
             <?php
             foreach ($breadcrumb as $value) {
-                ?><li class="<?=$value['active'] ? 'ccm-undroppable-search-item active' : 'ccm-droppable-search-item'?>" data-collection-id="<?=isset($value['id']) ? $value['id'] : ''?>"><?php
+                ?><li class="<?=$value['active'] ? 'ccm-undroppable-search-item active' : 'ccm-droppable-search-item'; ?>" data-collection-id="<?=isset($value['id']) ? $value['id'] : ''; ?>"><?php
                 if (isset($value['children'])) {
                     ?><span class="dropdown">
                     <button type="button" class="btn btn-default btn-xs" data-toggle="dropdown">
-                        <?=$value['name']?>
+                        <?=$value['name']; ?>
```

Comment:
There's not really a reason to add a semi colon here, if you want to adjust these to match our code style you'd do `<?= $value['name'] ?>`

Reply:
Yeah that was Phpstorm, it run a PHP CS Fixer thing, forgot to fix that one, I will look for more of those I might missed.

Reply:
Looks like our phpcs setup forces these semicolons at the moment, do whatever makes phpcs happy at this point

Reply:
My PhpStorm suggests me not to use the semicolons, but PHP CS adds the semicolon. IMO, we should update the PHP CS config.

Reply:
Sounds like @mlocati recommends we use the `c5:phpcs` command instead of phpcs directly. So at this point we should disable phpcs integration in phpstorm

Reply:
@mesparza5 @KorvinSzanto about the problem with those semicolons: it's controlled by the [`semicolon_after_instruction` fixer](https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:semicolon_after_instruction), included in the [`@Symphony` fixer set](https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixerset:@Symfony) included in the root [`.php_cs.dist` file](https://github.com/concrete5/concrete5/blob/81ca4de81272551a0958a175a5448b6d54d53901/.php_cs.dist#L9).
That fixer is good for files that contain only PHP code (like classes in the `src` directory), but shouldn't be applied in view files.
That's why in https://github.com/concrete5/concrete5/pull/8519/ I removed the `.php_cs.dist` file (we should use the `c5:phpcs` CLI command), and [enabled the `semicolon_after_instruction` fixer for PHP-only files](https://github.com/concrete5/concrete5/pull/8519/commits/22497dad74ca1d62d222f2ef4620a5ee4cb7bdf5).